<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cNewElementWindow</class>
 <widget class="QDialog" name="cNewElementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>531</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphics_standard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Element Name : </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addTemplate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>352</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add standard template</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editImage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit image file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addPoint">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>352</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add point</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphics_user">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cNewElementWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cNewElementWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
